### ISSUE TITLE
`type2-runtime` version/commit should be informed in stdout

### ISSUE THREAD

**Issue by brunvonlope** [AppImage/appimagetool#80]
When using appimagetool we know its version seeing the output:
```
appimagetool, continuous build (git version feac857), build 185 built on 2024-11-24 19:31:44 UTC
```

However, it don't give any clue about what "version" (commit sha) of the runtime will be used:
```
Downloading runtime file from https://github.com/AppImage/type2-runtime/releases/download/continuous/runtime-x86_64
Downloaded runtime binary of size 696720
```

Right now, developers need to rely on date and time of the creation of the appimage to get an estimative of what version/commit of the runtime is being used, which is imprecise. So, having this info in stdout would be very useful for debugging.

**Comment by TheAssassin** [comment]
I'm not sure whether we include those data in the runtime in some portable way. Right now, we just don't have that information. Plus, continuous is a moving target anyway. We've always considered it stable, though. But if we come up with a nice way to transport the information, we can for sure display it in appimagetool.

In the future, we want to make actual releases of all components, then we can also, e.g., target the latest stable release.

**Comment by probonopd** [comment]
At the point in time when `appimagetool` uses the runtime file, it could run `--appimage-version` on the runtime file and print out that information.

**Comment by TheAssassin** [comment]
We should have an option to detect that information statically, i.e., without the need for a suitable runtime environment. An ARM runtime cannot be run on an AMD64 system, but the AppImage could be built normally.

**Comment by probonopd** [comment]
Good point.

**Comment by brunvonlope** [comment]
Running `--appimage-version` in appimages generated by appimagetool gives **no** output:

`AppImage runtime version: `

So we don't have  even a workaround.

appimages generates by go-appimagetool (e.g.: the own go-appimagetool .appimage) have proper output of commit sha but the runtime seems to be different (static-tools). So, probably there is something wrong with type2-runtime(?)

**Comment by probonopd** [comment]
Indeed, that's a bad bug. Opening an issue there. Thanks @brunvonlope
